src/main/java/org/owasp/esapi/logging/appender/LogPrefixAppender.java

```diff
@@ -91,6 +107,11 @@ public String appendTo(String logName, EventType eventType, String message) {
             }
         }
 
-        return String.format(RESULT_FORMAT, logPrefix.toString(), message);
+        if (logPrefix.toString().trim().isEmpty()) {
+            // if there isn't any log prefix we just send back the message without touching it
+            return message;
+        }
+
+        return String.format(RESULT_FORMAT, logPrefix.toString().trim(), message);
```

Comment:
I would feel better about this if it was a single return as a ternary:

```
String prefix = logPrefix.toString().trim();
return prefix.isEmpty() ? message : String.format(RESULT_FORMAT, prefix, message);
```

Reply:
I'm good with that.